akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraOffsetStore.scala

```diff
@@ -63,11 +76,13 @@ import akka.stream.alpakka.cassandra.scaladsl.CassandraSession
         s"CREATE KEYSPACE IF NOT EXISTS $keyspace WITH REPLICATION = { 'class' : 'SimpleStrategy','replication_factor':1 }")
       .flatMap(_ => session.executeDDL(s"""
         |CREATE TABLE IF NOT EXISTS $keyspace.$table (
-        |  projection_id text,
+        |  projection_name text,
+        |  partition int,
+        |  projection_key text,
         |  offset text,
         |  manifest text,
         |  last_updated timestamp,
-        |  PRIMARY KEY (projection_id))
+        |  PRIMARY KEY (projection_name, partition))
```

Comment:
`projection_key` should also be included

```
PRIMARY KEY ((projection_name, partition), projection_key)
```

note that the first two are within their own `()`

Reply:
:+1: